src/Microsoft.AspNet.Server.Kestrel/Http/Frame.cs

```diff
@@ -39,6 +40,12 @@ public abstract partial class Frame : FrameContext, IFrameControl
         private static readonly byte[] _bytesDate = Encoding.ASCII.GetBytes("Date: ");
         private static readonly byte[] _bytesEndHeaders = Encoding.ASCII.GetBytes("\r\n\r\n");
 
+        private static readonly Vector<byte> _vectorCRs = new Vector<byte>((byte)'\r');
```

Comment:
I'm guessing its because it has to replicate the byte 32 times for each of the Vector's entries. Would probably get a similar gain by initializing it with a repeated array instead; except the framework would probably add some bounds checks...